addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1134,9 +1134,12 @@ function listGroupToTable(editable) {
 function normalizeColors($editable) {
     const editable = $editable.get(0);
     for (const node of editable.querySelectorAll('[style*="rgb"]')) {
-        const rgbMatch = node.getAttribute('style').match(/rgb?\(([\d\.]*,?\s?){3,4}\)/g);
+        const rgbMatch = node.getAttribute('style').match(/rgb[a]*?\(([\d\.]*,?\s?){3,4}\)/g);
+        const bg = getComputedStyle(node).backgroundColor;
```

Comment:
I feel like `bg` is a bit too simple as a variable name on its own, especially when playing around `rgb` and `rgba` things. I feel like `bgColor` would be better. Another option, considering it's only used once, would be just to inline it and have: 
```js
        const bgRgbMatch = getComputedStyle(node).backgroundColor.match(/\((.*)\)/);
```
Though I guess it looks a little complicated... your call. Note that `bgRbgMatch` should probably be renamed `bgRgbaMatch`, right ?

Reply:
OK for `bgColor` and `bgRgbaMatch`.